src/components/CCPDashboard/CCPDashboardPage.tsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
-import { Box, Tabs, Tab, makeStyles, Typography } from '@material-ui/core';
+import {
+  Box,
+  Tabs,
+  Tab,
+  makeStyles,
+  Typography,
+  IconButton,
+  Badge,
+} from '@material-ui/core';
+import RefreshIcon from '@material-ui/icons/Refresh';
 import { RouteComponentProps } from 'react-router';
-import { useQuery } from '@apollo/react-hooks';
+import { useLocation } from 'react-router-dom';
+import { useQuery, useSubscription } from '@apollo/react-hooks';
```

Comment:
@wongcmichelle just wanted to confirm this shouldn't be an issue? i remember at some point we talked about not using `@apollo/react-hooks` and instead using a different (and potentially older?) equivalent?

Reply:
I don't recall that conversation :/ Using `@apollo/react-hooks` should be fine :)